MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,39 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.95.0 ➞ v0.96.0
+### snowflake_row_access_policy resource changes
+New fields:
+  - `show_output` field that holds the response from SHOW ROW ACCESS POLICIES.
+  - `describe_output` field that holds the response from DESCRIBE ROW ACCESS POLICY.
+
+#### *(breaking change)* Renamed fields in snowflake_row_access_policy resource
+Renamed fields:
+  - `row_access_expression` to `body`
+  - `signature` to `arguments`
+Please rename these fields in your configuration files. State will be migrated automatically.
+
+#### *(breaking change)* Adjusted behavior of arguments/signature
+Now, arguments are stored as a list, instead of a map. Please adjust that in your configs. State is migrated automatically. Also, this means that order of the items matters and may be adjusted.
+
+Argument names are now case sensitive.
+
+#### *(breaking change)* Adjusted behavior on changing name
+Previously, after changing `name` field, the resource was recreated. Now, the object is renamed with `RENAME TO`.
+
+#### *(breaking change)* Mitigating permadiff on `body`
```

Comment:
is it a breaking change or a bugfix?

Reply:
A breaking change because now spacing can affect body when spaces are semantically significant. Before this wasn't the case. Also, permadiff wasn't happening for all queries, so some users could see a regression here.